README.md

```diff
@@ -67,7 +67,7 @@ published to the Rapid7 Insight Platform every second.
 
 - `--no-logs` if you do not want logs to be published to the Rapid7 Insight Platform.
 
-- `--no-dockerEvents` if you do not want events to be
+- `--no-events` if you do not want events to be
```

Comment:
i think `docker-events` is clearer in this case. iirc this relates to log messages originiating from the creation/deletion/... of containers?

Reply:
Yup, administrative tasks done by docker.

I made this change because we refer to them everywhere in here as events, but just for the flag we use `dockerEvents`

I'd prefer drop a comment in the README to explain what they are, or I can make the entire thing consistent with `dockerEvents`, what you prefer?

Reply:
docker-events, definitely :) although let's not do camel casing in cli arguments -- whatever we change this too though let's make sure the old one keeps working. let's not unnecessarily break configuration options for little benefit. changing them is fine, but a "redirect" would be nice

Reply:
or even a deprecation notice of sorts; i.e. "process started with --dockerEvents. this has been renamed to --docker-events and will stop working in future." but keep supporting it for now

Reply:
> whatever we change this too though let's make sure the old one keeps working

just makes the package far more complicated than it needs to be with multiple ways to specify the same thing
we're bumping major so it's explicit that `hey this release breaks something`

i'm more than happy to deal with the customers who complain about this